<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowDialog</class>
 <widget class="QDialog" name="MainWindowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(32, 32, 32);
}</string>
  </property>
  <widget class="QPushButton" name="plusMinusBtn">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>447</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zeroBtn">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>447</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="commaBtn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>447</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equalsBtn">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>447</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(138, 22, 155);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(38, 6, 43);
}

QPushButton::hover {
	background-color: rgb(115, 18, 130);
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="threeBtn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>392</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="oneBtn">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>392</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twoBtn">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>392</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plusBtn">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>392</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
	border-radius: 4px;
	font: 700 18pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
	background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fiveBtn">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>337</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sixBtn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>337</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fourBtn">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>337</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minusBtn">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>337</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
	border-radius: 4px;
	font: 700 18pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>–</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eightBtn">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>282</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moltiplicationBtn">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>282</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
	border-radius: 4px;
	font: 700 18pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nineBtn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>282</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sevenBtn">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>282</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(89, 89, 89);
	border-radius: 4px;
	font: 700 20pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(50, 50, 50);
}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="X2_Btn">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>227</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
	border-radius: 4px;
	font: 700 18pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>x²</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DivisionBtn">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>227</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
	border-radius: 4px;
	font: 700 18pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>÷</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rootBtn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>227</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
	border-radius: 4px;
	font: 700 18pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>²√x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="X_Btn">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>227</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
	border-radius: 4px;
	font: 700 18pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>1/x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CE_Btn">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>172</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
	border-radius: 4px;
	font: 700 18pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>CE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DEL_Btn">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>172</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
	border-radius: 4px;
	font: 700 18pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>DEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="C_Btn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>172</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
	border-radius: 4px;
	font: 700 18pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="percentageBtn">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>172</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
	border-radius: 4px;
	font: 700 18pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
}

QPushButton::hover {
background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numbersBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>55</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 700 22pt &quot;Segoe UI&quot;;
	color: rgb(255, 255, 255);
	padding-left: 15px;
	background-color: rgb(32, 32, 32);
	border: 0px solid rgb(32, 32, 32);
	letter-spacing: 2px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imgBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(32, 32, 32);
border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="historyLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>326</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(32, 32, 32);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
